src/types.ts

```diff
@@ -6,22 +6,40 @@ export interface Options {
    * Relative paths to the directory to search for components.
    * @default 'src/components'
    */
-  dirs: string | string[]
+  include_dirs: string | string[]
+
+  /**
+   * Directory names that are ignored as Component names.
+   * Only works if Foldernames are allowed
+   * @default "['global', 'partials']"
+   */ 
+  namespaces: string[]
+
   /**
    * Valid file extensions for components.
    * @default ['vue']
    */
   extensions: string | string[]
+
   /**
    * Search for subdirectories
    * @default true
    */
   deep: boolean
+
+  /**
+   * Allow directories as Names for components
+   * @default false
+   */
+
+  allowFolderNames: boolean
```

Comment:
Not very share about the naming. But `allow` something does not sounds good to an optional feature. Maybe:

- folderToPrefix
- folderNamespace

What do you think?

Reply:
Yeah right, both options i didnt like!
folderNamespace would be good